### ISSUE TITLE
useDayjs composable return any

### ISSUE THREAD

**Issue by TouchSek** [fumeapp/dayjs#30]
Ref: 
<img width="463" alt="image" src="https://github.com/fumeapp/dayjs/assets/53075504/a7c7a969-259c-4020-9090-53b19223fc6f">
It should be kind of this.
```ts
import dayjs from "dayjs";

export declare const useDayjs: () => typeof dayjs;
```

**Comment by GaborTorma** [comment]
fixed in #31

**Comment by acidjazz** [comment]
https://github.com/fumeapp/dayjs/releases/tag/v2.1.9
